### PR TITLE
Pointing mention-bot.js file to dir

### PR DESCRIPTION
**Pull request by KrishMunot**

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
@KrishMunot updated the pull request.

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ Alternatively, click the button below:
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
-If you would like the mention-bot to function on private repositories, set the `GITHUB_USER` and `GITHUB_PASSWORD` environment variables. You must disable two-factor authentication or you will receive a console log like this: `Login to ${USERNAME} failed`.
+If you would like the mention-bot to function on private repositories, set the `GITHUB_USERNAME` and `GITHUB_PASSWORD` environment variables. You must disable two-factor authentication or you will receive a console log like this: `Login to ${USERNAME} failed`.
```
Comment (vjeux) [inline README.md]: It's actually GITHUB_USER. It's not great but that's the name :(

File: README.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ If you use `http` protocol, the config section like this:
 
 ## Programmatic API
 
-When you require `mention-bot` you will get all the functions exposed by `mention-bot.js` module. You are expected to manage your own server and also connection to the github repository.
+When you require `mention-bot` you will get all the functions exposed by [`mention-bot.js`](https://github.com/facebook/mention-bot/blob/master/mention-bot.js) module. You are expected to manage your own server and also connection to the github repository.
```
Comment (vjeux) [inline README.md]: Sorry for the super long wait, if you are still interested, could you send this change, it is good!

Comment (KrishMunot) [inline README.md]: Sure! I just made an update to the [PR](https://github.com/facebook/mention-bot/pull/116/files) to make it fit for the merge :)

**Comment by facebook-github-bot-2** [comment]
@KrishMunot updated the pull request.

**Pull request merged**